src/components/productCard/productCard.jsx

```diff
@@ -3,7 +3,8 @@ import { cartSolid } from "../../assets/images/index";
 import { BiHeartFill, BiXLg, BiStarFill } from "../../assets/icons/Icons";
 import { useProducts } from "../../context/product-context";
 import axios from "axios";
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuth } from "../../context/auth-context";
```

Comment:
Try barrel exports.

Check this out: https://blog.justmyrealname.dev/barrel-exporting-in-react

Reply:
Done. Thanks for pointing out!